### ISSUE TITLE
Version on Atmosphere not working

### ISSUE THREAD

**Issue by dalgard** [Sewdn/meteor-collection-behaviours#19]
I've been banging my head into a wall the last couple of days. It turned out that the version of `collection-behaviors` that got installed with `meteor add` was 0.1.5, which has a constraint for `collection-hooks` 0.7.3 – which doesn't work.

I have three questions:
1. Wouldn't it be better to not use a hard constraint?
2. How come Atmosphere doesn't have the newest version? Does it mean that I can never trust Atmosphere again?
3. Can you solve this or help me solve it?

**Comment by rhyslbw** [comment]
@Sewdn, are you able to resolve this issue with Atmosphere? The workaround of downloading the repo into my packages directory isn't working

**Comment by dalgard** [comment]
+1

@rhyslbw, the workaround is working fine for me, but you still need to add the package name to your `.meteor/packages` file.

**Comment by pmuens** [comment]
You need to run `meteor add sewdn:collection-behaviours` after you've copied the package into the packages-Folder.

**Comment by rhyslbw** [comment]
Where is 0.2.0? The version on the master branch here in this repo is 0.1.6

**Comment by pmuens** [comment]
Hmm. The version I see here in the master-Branch is Version 0.2.0.

**Comment by rhyslbw** [comment]
Oh right, I was looking at the Releases. I have cloned the repo into my packages directory and then run the add command, but it still pulls in 0.1.6. 

Do packages need to be resubmitted after an update?

**Comment by dalgard** [comment]
I'm pretty sure they do – there's no integration between Atmosphere and GitHub.

**Comment by pmuens** [comment]
Yes, they have to be resubmitted when the maintainer changes the code.

**Comment by rhyslbw** [comment]
http://autopublish.meteor.com/howitworks

Be good to see this project get some traction
